core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/DetailedQcStatus.java

```diff
@@ -16,4 +16,8 @@ public interface DetailedQcStatus extends Deletable, Identifiable, Serializable,
 
   void setNoteRequired(Boolean noteRequired);
 
+  Boolean getArchived();
+
+  void setArchived(Boolean archived);
```

Comment:
Use `boolean` rather than `Boolean` since this should not be nullable (`noteRequired` field above is a poor example unfortunately)